### ISSUE TITLE
Block "Who is Guillaume" not accessible when using the keyboard

### ISSUE THREAD

**Issue by EmmanuelDemey** [EmmanuelDemey/GlobalIssueTracker#44]
# Informations générales

* URL du site : https://codewithguillaume.com/

# Descriptions

<img width="1197" alt="image" src="https://user-images.githubusercontent.com/555768/235449547-5aeb8270-a02c-4274-9569-6fef9dbb212c.png">

# Quelques Liens
